src/umpire/util/Macros.hpp

```diff
@@ -86,11 +87,15 @@
 
 #define UMPIRE_USE_VAR(x) static_cast<void>(x)
 
+
 #define UMPIRE_ERROR( msg )                                        \
 {                                                                  \
   UMPIRE_LOG(Error, msg);                                          \
   std::ostringstream umpire_oss_error;                             \
   umpire_oss_error << " " << __func__ << " " << msg;               \
+  umpire::log().flush();                                           \
```

Comment:
Can you wrap this up in a function in io.cpp?

Reply:
Done.